docs/src/main/tut/datatypes/averaged_value.md

```diff
@@ -8,8 +8,16 @@ scaladoc: "#com.twitter.algebird.AveragedValue"
 
 # Averaged Value
 
-### Documentation Help
+## Algebraic Properties
+
+## Usage Examples
```

Comment:
Seems like just showing some example of getting the average of a bunch of Longs or Doubles would be fine.